product.json

```diff
@@ -3,5 +3,6 @@
 	"nameLong": "Code [OSS Build]",
 	"win32MutexName": "vscodeoss",
 	"licenseUrl": "https://github.com/Microsoft/vscode/blob/master/LICENSE.txt",
-	"darwinBundleIdentifier": "com.visualstudio.code.oss"
+	"darwinBundleIdentifier": "com.visualstudio.code.oss",
+	"welcomePage": "http://go.microsoft.com/fwlink/?LinkId=723048"
```

Comment:
@SofianHn this has to go into the VS Code distro repo: https://github.com/Microsoft/vscode-distro